test/objectstorage_tests/go.mod

```diff
@@ -9,13 +9,41 @@ require (
 	golang.org/x/oauth2 v0.5.0
 )
 
+require (
```

Comment:
better to use `document` naming everywhere
e.g. `module github.com/codenotary/immudb/test/document_storage_tests`

even in the folders e.g. `tests/document_storage_tests` and `document_storage_tests`

Reply:
This was already like that, but I'm changing it.